external/kafka-0-10-sql/pom.xml

```diff
@@ -92,6 +92,18 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
```

Comment:
This can be added to the root pom since MiniKDC can be useful for all of the peoples who implements secure tests but since only Kafka connector uses it I've left it here for now.

Reply:
I am working on SparkThriftServer. also need miniKDC.

Reply:
Good to hear! Since test deps are not transitive the mentioned dependency has to be added when this is merged.

Reply:
Moved.